prqlc/prqlc/Cargo.toml

```diff
@@ -54,6 +54,7 @@ sqlformat = "0.2.3"
 sqlparser = {version = "0.46.0", features = ["serde"]}
 strum = {version = "0.26.2", features = ["std", "derive"]}
 strum_macros = "0.26.2"
+clap-verbosity-flag = "2.2.0"
```

Comment:
Ah, and we should move this down to the next section, allow it to be optional, and add it to the cli feature; otherwise it'll interfere with the wasm compilation...

Reply:
fixed 👍